### PR TITLE
[Cloudflare] Fix SyntaxError due to recent js challenge changes.

### PR DESCRIPTION
**Pull request by DitFranXX**
From Anorov/cloudflare-scrape#193
Related issues: inorichi/tachiyomi-extensions#894

~~This PR has Line Ending changes. Sorry. (That's why i spammed force push)~~
Turned off AutoCRLF xD

### CODE REVIEW

**Comment by CarlosEsco** [comment]
@inorichi can you review

**Comment by DitFranXX** [comment]
Build Failed due to `Slice:v1.2` (https://jitpack.io/com/github/mthli/Slice/v1.2/Slice-v1.2.aar)

**Comment by CarlosEsco** [comment]
@ditfranxx anyway you can clean up the line breaks. So it doesn't show a whole new file being replaced lol

**Comment by inorichi** [comment]
Thanks. Let me some time to check the failing build before I merge.

**Comment by DitFranXX** [comment]
Build is successful because I pushed CRLF commits after Jitpack trigger the (re)build Slice `v1.2`  release

File: app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt [hunk 1]
```diff
@@ -1,11 +1,7 @@
 package eu.kanade.tachiyomi.network
 
 import com.squareup.duktape.Duktape
-import okhttp3.CacheControl
-import okhttp3.HttpUrl
-import okhttp3.Interceptor
-import okhttp3.Request
-import okhttp3.Response
+import okhttp3.*
```
Comment (MclaughlinSteve) [inline app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Is there a standard in this project for whether to do explicit imports vs wildcard imports?

Comment (inorichi) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Not really on master, but on 1.x I'm forcing explicit imports.

Comment (DitFranXX) [reply app/src/main/java/eu/kanade/tachiyomi/network/CloudflareInterceptor.kt]: Caused by IDEA's imports optimize.

**Pull request merged**